src/wallet/balance.go

```diff
@@ -21,6 +21,15 @@ type Balance struct {
 	Hours uint64 `json:"hours"`
 }
 
+// BalanceResult contains the overall balance as well as of individual addresses
```

Comment:
These changes aren't needed in wallet package, the structs can be in the gui package